src/backend/common/data/commands/players.py

```diff
@@ -363,4 +363,28 @@ async def handle(self, db_wrapper, s3_wrapper):
                 rowcount = cursor.rowcount
                 if rowcount == 0:
                     raise Problem("Name edit request not found", status=404)
-                await db.commit()
\ No newline at end of file
+                await db.commit()
+
+@dataclass
+class GetPlayerTransferHistoryCommand(Command[PlayerTransferHistory]):
+    player_id: int
+
+    async def handle(self, db_wrapper, s3_wrapper):
+        history: list = []
+        async with db_wrapper.connect(readonly=True) as db:
+            async with db.execute('''SELECT t.id, t.name as "team_name", tm.join_date, tm.leave_date, tr.name as "roster_name"
+                FROM team_members as tm
+                JOIN team_rosters as tr
+                ON tm.roster_id = tr.id
+                JOIN teams as t
+                ON t.id = tr.team_id
+                WHERE player_id = ?
+                ORDER BY tm.join_date ASC;''',
+                (self.player_id,)) as cursor:
+                rows = await cursor.fetchall()
+                for row in rows:
+                    team_id, team_name, join_date, leave_date, roster_name = row
+                    history.append(PlayerTransferItem(team_id, team_name, join_date, leave_date, roster_name))
+                results = PlayerTransferHistory(history)
+                return results
```

Comment:
This command should require and return a game/mode parameter, otherwise it is difficult to make sense of any of the transfer history when viewing the page. We should also use roster name instead of team name if it exists to differentiate between different rosters of the same team.